updating/update-using-custom-machine-config-pools.adoc

```diff
@@ -0,0 +1,167 @@
+[id="update-using-custom-machine-config-pools"]
+= Use of customized machine config pools during update
+include::modules/common-attributes.adoc[]
+:context: update-using-custom-machine-config-pools
+
+toc::[]
+
+In OpenShift V4 nodes are not considered individually. Nodes are considered in groups i.e. machine config pools(MCP). In a typical OCP cluster we have two MCPs. The MCP for the control plane is called master pool and other group which have worker nodes is called worker pool. During an OpenShift update all pools are updated concurrently.The MCP update process involves updating the machines to the new OS version(if required) along with updating some daemonsets and reboot it to comeplete the process. The MCP gives us the ability to pause a pool and it will make sure that the OS is not updated/rebooted during the update process. We can create more than one customized machine config pools out of the the worker pool and will give us a lot of control on the sequence in which the worker nodes are updated.
```

Comment:
nit: `concurrently.The` -> `concurrently. The` (adding a missing space after the period).  Same for `version(if` -> `version (if`.

Also:

> We can create more than one customized machine config pools out of the the worker pool and will give us a lot of control on the sequence in which the worker nodes are updated.

to:

> We can create more than one customized machine config pool out of the worker pool, which will give us a more control over the sequence in which the compute nodes are updated.

or some such.

Reply:
Fixed in https://github.com/openshift/openshift-docs/commit/c27e14a1b4785cc8704f11d191797f198c10f027